README.md

```diff
@@ -1,69 +1,20 @@
 # percy-webdriverio
-
+[![Package Status](https://img.shields.io/npm/v/@percy-io/percy-webdriverio.svg)](https://www.npmjs.com/package/@percy-io/percy-webdriverio)
 [![Build Status](https://travis-ci.org/percy/percy-webdriverio.svg?branch=master)](https://travis-ci.org/percy/percy-webdriverio)
 
-Webdriver plugin for percy.io support
+**@percy-io/percy-webdriverio** adds [Percy](https://percy.io) visual testing and review to your [**WebdriverIO**](http://webdriver.io/) tests.
 
-# Hello world:
+#### Docs here: [https://percy.io/docs/clients/javascript/webdriverio](https://percy.io/docs/clients/javascript/webdriverio)
```

Comment:
Being consistent with our other repos, provide minimal information here, and link to our docs page.